File: MultiFlex/API/JavaRestApi/src/main/java/org/acme/DTO/SupplierDto.java
package org.acme.DTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class SupplierDto implements Serializable {
    private Integer id;

    private String name;

    private String link;

    private Integer deliveryTime;

    private List<Integer> ware_ids;
}

File: MultiFlex/Frontend/Multiflex.Frontend/MultiFlex.Frontend.WebApp/Models/Dto/SupplierDto.cs
using System.Collections.Generic;

namespace Multiflex.Frontend.WebApp.Models
{
    public class SupplierDto : IdentityModel
    {
        public string name { get; set; }
        public string link { get; set; }
        public int deliveryTime { get; set; }
        public List<int> ware_ids { get; set; }
    }
}
